recipes-core/aziotd/aziotd_1.4.6.bb

```diff
@@ -264,8 +264,8 @@ SRC_URI += " \
     crate://crates.io/ws2_32-sys/0.2.1 \
 "
 
-LIC_FILES_CHKSUM=" \
-file://LICENSE;md5=4f9c2c296f77b3096b6c11a16fa7c66e \
+LIC_FILES_CHKSUM = " \
+    file://LICENSE;md5=4f9c2c296f77b3096b6c11a16fa7c66e \
```

Comment:
is this tab space make any changes?

Reply:
Same as previous comment, just trying to reuse what `cargo bitbake` generates as much as possible. I assume a minor change like this won't make a difference (although I'm not 100% sure).

Reply:
@ggjjj If the build succeeds, is that validation enough?

Reply:
@ggjjj Take a look at the SRC_URI append statement right before LIC_FILES_CHKSUM assignment; it does the same thing. It has multiple lines, and the element on each line is indented four spaces.